fusor-ember-cli/app/controllers/review/progress/log.js

```diff
@@ -39,18 +41,18 @@ export default Ember.Controller.extend(NeedsDeploymentMixin, {
     Ember.run.once(this, () => this.send('updateDisplayedLog'));
   }),
 
-  isSearchActive:Ember.computed('searchLogString', function() {
+  isSearchActive: Ember.computed('searchLogString', function() {
     return !!this.get('searchLogString');
   }),
 
-  logTypeChanged: function() {
-    this.set('displayedLogHtml', '');
-    this.set('newEntries', []);
-    // run later to allow the dropdown to close and log to clear before doing the real work
-    Ember.run.later(this, () => { this.send('changeLogType'); });
-  }.observes('logType'),
-
   actions: {
+    logTypeChanged: function() {
+      this.set('displayedLogHtml', '');
+      this.set('newEntries', []);
+      // run later to allow the dropdown to close and log to clear before doing the real work
+      Ember.run.later(this, () => { this.send('changeLogType'); });
```

Comment:
Does `run.later` expect a millisecond timeout? (or use a default)?
https://github.com/emberjs/ember.js/blob/v1.13.13/packages/ember-metal/lib/run_loop.js#L299-L324

I wonder if `Ember.run.scheduleOnce('afterRender', /***/)` is better in this case since it more explicitly schedules the work post render?

https://guides.emberjs.com/v1.10.0/understanding-ember/run-loop/#toc_what-happens-in-these-queues